### DISCUSSION TITLE
How can I create a lighting invoice using python API?

### DISCUSSION THREAD

**Discussion by mightyroy** [btcpayserver/btcpayserver#4090] [Technical Support]
There's documentation for invoice creation for on chain payments with the python API but I couldn't find any on how to generate a lightning invoice and check lightning payment status. I'm building Point-of-sale terminal for my restaurant. Can anyone help?

**Comment by bolatovumar** [comment]
@mightyroy what Python API documentation are you referring to? Do you have a link?
